backend/src/main/java/com/jobwait/security/UUIDJumbler.java

```diff
@@ -0,0 +1,33 @@
+package com.jobwait.security;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class UUIDJumbler {
+    private static String bytesToHex(byte[] hash) {
+        StringBuilder hexString = new StringBuilder(2 * hash.length);
+        for (int i = 0; i < hash.length; i++) {
+            String hex = Integer.toHexString(0xff & hash[i]);
+            if (hex.length() == 1) {
+                hexString.append('0');
+            }
+            hexString.append(hex);
+        }
+        return hexString.toString();
+    }
```

Comment:
Might have been better to go with a library for this (https://stackoverflow.com/a/9655275) so we don't have to understand this jumbler, but if you already wrote it and know it works, it should be fine.

Reply:
I did consider this but then got a bit lazy to add in a whole library for a simple function. With regards to how it works ehmm I just copied the code from https://www.baeldung.com/sha-256-hashing-java which seems very trustworthy. If you are worried we can use the Apache solution you linked.

Reply:
If you're not worried, I'm not worried. If it's done, let's keep it done